src/Tasks.UnitTests/GenerateResource_Tests.cs

```diff
@@ -3135,6 +3135,55 @@ public void ResxValueNewlines(string newline)
                 }
             }
         }
+
+        [Fact]
+        public void RebuildsInPresenceOfFileRefWithWindowsPath()
```

Comment:
nit: a better name would be `GenerateResourceShouldNotRebuildInPresenceOfFileRefWithWindowsPath`. Either way, the name should specify that a rebuild should not happen.